.git-blame-ignore-revs

```diff
@@ -1,6 +1,4 @@
-# .git-blame-ignore-revs
```

Comment:
Don't we want to keep the old hashes and we will need to update the new list after the PR is merged into main.

Reply:
Fixed this, and added the new commit hash.